test/e2e/operator_groups_e2e_test.go

```diff
@@ -127,21 +127,30 @@ var _ = Describe("Operator Group", func() {
 		}
 		_, err = crc.OperatorsV1().OperatorGroups(opGroupNamespace).Create(context.TODO(), &operatorGroup, metav1.CreateOptions{})
 		require.NoError(GinkgoT(), err)
-		expectedOperatorGroupStatus := v1.OperatorGroupStatus{
-			Namespaces: []string{opGroupNamespace, createdOtherNamespace.GetName()},
-		}
+
+		// fetched namespaces might be in any order
+		namespaces := map[string]bool{}
```

Comment:
If you use sets.Set[string], it makes comparisons easier

Reply:
This is a pretty simple use; not sure changing to `sets` will improve this.

Reply:
The SymmetricDifference helper is really useful specifically for the comparison you're making in this test to be clear on the diff. Not a big deal for N=2 but much more useful as the set gets larger.

As an aside it would also make the semantics clearer - what would it mean for an entry to be in the map but `false`?

Reply:
Given the quantity and the very isolated, localized use case, not sure it's worth making the change.

@stevekuznetsov, I know what you're saying, but again, not sure it matters here.

Reply:
Sure, no worries - this is fine. I tend to think of it in the other way - we've got a great helpful library that gives you the exact output you want to show devs when the test fails, and using it is no trouble at all - why not? :)